src/main/java/com/transferwise/openbanking/client/oauth/domain/Scope.java

```diff
@@ -7,7 +7,8 @@
 public enum Scope {
     ACCOUNTS("accounts"),
     PAYMENTS("payments"),
-    OPENID("openid");
+    OPENID("openid"),
+    COPPHASE2("name-verification");
```

Comment:
^ not sure if it's correct, the step to retrieve the token in the simulator is passing though